src/store/config.js

```diff
@@ -43,6 +43,7 @@ const mutations = {
     state.openIdConnect = config.openIdConnect
     state.rootFolder = config.rootFolder === undefined ? '/' : config.rootFolder
     state.state = config.state === undefined ? 'working' : config.state
+    state.applications = config.applications === undefined ? [] : config.applications
```

Comment:
state.applications = config.applications || []

Reply:
would have the same result. I just copied the pattern from above so they all look like they're dancing the same aligned dance